src/main/resources/view/HelpWindow.fxml

```diff
@@ -10,7 +10,7 @@
 
 <!-- TODO: set a more appropriate initial size -->
 
-<fx:root maximized="true" title="Help" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/11" xmlns:fx="http://javafx.com/fxml/1">
+<fx:root maximized="false" title="Help" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/11" xmlns:fx="http://javafx.com/fxml/1">
```

Comment:
The "maximized" attribute can be removed because "false" is the default. Gluon SceneBuilder would have removed it as well.

Reply:
Yeah somehow scene builder now absolutely refuses to open the FXML files. I might need to check if it's a version issue so we can peg a version of scene builder for students. I'll go check it out for sure.

Reply:
fwiw scene builder works for me. I'm using the version 11 one.

Reply:
I'll go make a new PR that bumps JavaFX to version 11.0.1 before I go peg a version of Scene Builder I guess.